### PR TITLE
Utest, Methods, quantitative analysis of Methods A-I

### PR DESCRIPTION
**Pull request by gjaeger**
subtask of issue #165

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +30,13 @@ matrix:
...
+    - choco install python -- version 3.7.4
+    - python -m pip install --upgrade pip
+    env: PATH=/c/Python37:/c/Python37/Scripts:%PATH
 
 install:
     - source .travis_install.bash
+    - pip install --user --upgrade pip
+    - pip install --user numpy
+    - pip install --user scipy
+    - pip install --user logging
```
Comment (chraibi) [inline .travis.yml]: these installs can go in the script `travis_install.bash`

File: .travis.yml [hunk 2]
```diff
@@ -29,6 +30,13 @@ matrix:
 
 before_install:
     - source .travis_before_install.bash
+    - choco install python -- version 3.7.4
+    - python -m pip install --upgrade pip
+    env: PATH=/c/Python37:/c/Python37/Scripts:%PATH
```
Comment (chraibi) [inline .travis.yml]: these can go in .travis_before_install.bash

File: Utest/Methods/runtest_Method_C.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    velocity_exit = 0
+else:
+    logging.info('--> velocity: different dist (reject H0) -- p-value: {0:.4f}'.format(velocity_p_value))
+    velocity_exit = 1
+
+if (velocity_exit == 0) and (density_exit == 0):
+    logging.info('Method C: Test passed successfully.')
+    sys.exit(0)
+else:
+    logging.info('Method C: Test passed successfully.')
```
Comment (chraibi) [inline Utest/Methods/runtest_Method_C.py]: Test fails!

Comment (gjaeger) [reply Utest/Methods/runtest_Method_C.py]: Modified and implemented for all methods

File: .travis.yml (hunk 2 above)
Comment (gjaeger) [reply .travis.yml]: Seems to work with commit 9bb4be6.

File: .travis.yml (hunk 1 above)
Comment (gjaeger) [reply .travis.yml]: Seems to work with commit 9bb4be6.

**Comment by gjaeger** [comment]
> This should be ok. Please have a look at my comments. There are changes required..

Seems to work with commit 9bb4be6.

File: Utest/Methods/runtest_Method_B.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+import numpy as np
+from scipy.stats import ks_2samp
+
+import logging
+
+logging.basicConfig(
+    filename = "Utest_Methods.log",
+    level    = logging.DEBUG,
+    style    = "{",
+    format   = "{asctime} \t {levelname:8} \t {message} \t",
```
Comment (chraibi) [inline Utest/Methods/runtest_Method_B.py]: This works better with travis:
```
logging.basicConfig(
    filename = "Utest_Method_C.log",
    level    = logging.DEBUG,
    format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s',
    datefmt='%Y-%m-%d %H:%M:%S')
```

File: Utest/Methods/runtest_Method_A.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    style    = "{",
+    format   = "{asctime} \t {levelname:8} \t {message} \t",
+    datefmt='%Y-%m-%d %H:%M:%S')
+
+logging.info("===== Method A =========================")
+
+data_1_filename = 'reference_data_Method_A_Flow_NT.dat'
+data_1 = np.loadtxt(data_1_filename)
+
+data_2_filename = './Output_Methods/Fundamental_Diagram/FlowVelocity/Flow_NT_traj_Methods.txt_id_2.dat'
```
Comment (chraibi) [inline Utest/Methods/runtest_Method_A.py]: Why don't you use `reference_data_Method_A_Flow_NT.dat `?

Comment (gjaeger) [reply Utest/Methods/runtest_Method_A.py]: solved

File: Utest/Methods/runtest_Method_B.py (hunk 4 above)
Comment (gjaeger) [reply Utest/Methods/runtest_Method_B.py]: Considered in commit dfae253.

**Pull request merged**